opentelemetry-sdk/tests/metrics/integration_test/test_time_align.py

```diff
@@ -56,11 +63,11 @@ def test_time_align_cumulative(self):
             .data.data_points
         )
 
-        self.assertEqual(
+        self.assertLess(
```

Comment:
This is failing because values are equal:
https://github.com/open-telemetry/opentelemetry-python/actions/runs/10480926316/job/29029511296?pr=4137